pipeline/recorder/src/main/scala/uk/ac/wellcome/platform/recorder/Main.scala

```diff
@@ -3,32 +3,30 @@ package uk.ac.wellcome.platform.recorder
 import akka.actor.ActorSystem
 import akka.stream.ActorMaterializer
 import com.typesafe.config.Config
-import uk.ac.wellcome.json.JsonUtil._
-import uk.ac.wellcome.messaging.typesafe.{MessagingBuilder, SNSBuilder}
-import uk.ac.wellcome.models.work.internal.TransformedBaseWork
-import uk.ac.wellcome.platform.recorder.services.RecorderWorkerService
-import uk.ac.wellcome.storage.typesafe.VHSBuilder
-import uk.ac.wellcome.storage.vhs.EmptyMetadata
+
 import uk.ac.wellcome.typesafe.WellcomeTypesafeApp
 import uk.ac.wellcome.typesafe.config.builders.AkkaBuilder
+import uk.ac.wellcome.models.work.internal.TransformedBaseWork
+import uk.ac.wellcome.platform.recorder.services.RecorderWorkerService
+import uk.ac.wellcome.platform.recorder.Implicits._
 
-import scala.concurrent.ExecutionContext
+import uk.ac.wellcome.messaging.typesafe.SNSBuilder
+import uk.ac.wellcome.bigmessaging.typesafe.{BigMessagingBuilder, VHSBuilder}
 
 object Main extends WellcomeTypesafeApp {
+
   runWithConfig { config: Config =>
     implicit val actorSystem: ActorSystem =
       AkkaBuilder.buildActorSystem()
-    implicit val executionContext: ExecutionContext =
-      AkkaBuilder.buildExecutionContext()
     implicit val materializer: ActorMaterializer =
       AkkaBuilder.buildActorMaterializer()
 
     new RecorderWorkerService(
-      versionedHybridStore =
-        VHSBuilder.buildVHS[TransformedBaseWork, EmptyMetadata](config),
+      store = VHSBuilder.build[TransformedBaseWork](config),
       messageStream =
-        MessagingBuilder.buildMessageStream[TransformedBaseWork](config),
-      snsWriter = SNSBuilder.buildSNSWriter(config)
+        BigMessagingBuilder.buildMessageStream[TransformedBaseWork](config),
+      msgSender = SNSBuilder
+        .buildSNSMessageSender(config, namespace = "dunno", subject = "dunno")
```

Comment:
We use the `namespace` parameter if we have multiple instances of an SNSMessageSender in a single app, and we need to provide multiple topics. Here you can leave it empty and/or omit it, because this app only has one message sender.

The `subject` parameter isn't very important, just useful for debugging. We tend to use something like `"Sent from the recorder"` so we can trace the source of a message later if necessary.